Makefile

```diff
@@ -0,0 +1,7 @@
+install:
```

Comment:
Minor: if there's no build step, I tend to add something like:

```
.PHONY: all
all:
	@echo "No build step."
```

at the top so that (1) a `make all` still Just Works in an automated context, and (2) a pure `make` won't actually do a `make install`.

Let's maybe also add a `.PHONY` for `install`?

Reply:
Thanks, excellent feedback!  Updated :arrow_down: